aten/src/ATen/core/dispatch/DispatchTable.h

```diff
@@ -189,11 +189,10 @@ class DispatchTable final {
    * @param stack Stack with arguments to invoke the kernel function with
    * @return Kernel function pointing to the right kernel for the given arguments.
    */
-   const KernelFunction& lookupBoxed(const Stack* stack) const {
-     return lookup_([&] () -> c10::optional<TensorTypeId> {
-       return dispatch_strategy_.get_dispatch_key_boxed(stack);
-     });
-   }
+  const KernelFunction& lookupBoxed(const Stack* stack) const {
+    c10::optional<TensorTypeId> dispatchKey = dispatch_strategy_.get_dispatch_key_boxed(stack);
+    return lookup_(dispatchKey);
+  }
```

Comment:
Interesting. Do you know what happened here?

Reply:
This one wasn't the important one. The `lookupUnboxed` and `get_dispatch_key_unboxed` was. They're templates and if it's not short enough to be inlined, they get instantiated for each operator. Same for the lambda inside `lookupUnboxed` which is also a function that is instantiated for each op if it isn't inlined.